gravitee-apim-console-webui/src/management/api/creation/v4/api-creation-v4.component.ts

```diff
@@ -87,7 +79,7 @@ export class ApiCreationV4Component implements OnInit, OnDestroy {
       };
     });
     // Initialize stepper to step 0
-    this.stepper.goToStep(0);
+    this.stepper.goToStepLabel('API Metadata');
```

Comment:
This will always be the first step in the list, right? Maybe a way to avoid passing a string related to the copy?

Reply:
Yes i can make the observable emit the fist step once. i can do it in dedicated commit